flask_rebar/swagger_generation/swagger_generator_v3.py

```diff
@@ -119,7 +140,7 @@ def generate(self, registry, host=None, sort_keys=True):
         if self.tags:
             swagger[sw.tags] = [tag.as_swagger() for tag in self.tags]
 
-        servers = self.servers or []
+        servers = list(self.servers or [])
```

Comment:
can't `append()` to a `Sequence`

Reply:
It would probably be better to type `self.servers` as a list and convert the provided value in the `__init__` rather than creating a new list every time `generate` is called.